Makefile

```diff
@@ -1,4 +1,4 @@
-SERVER_CONFIG = config/kinto.ini
+SERVER_CONFIG = config/example.ini
```

Comment:
Do we expect kinto-dist users to modify `example.ini` in-place or copy it to `config/kinto.ini` and edit it there? I'd expect the second, which means I'd expect this to remain `config/kinto.ini` and have it error out if there was no config there.

You might also want to put `config/kinto.ini` in `.gitignore`.